docs/usage/faq.md

```diff
@@ -409,3 +410,12 @@ importlib.invalidate_caches() # Make sure Python notices the new .py file
 from mymodule import hello
 hello()
 ```
+
+## Why the changes I made in IndexDB doesn't persist?
+
+Unlike other filesystems, indexDB (pyodide.FS.filesystem.IDBFS) is an asynchronous filesystem.
+This is because browsers offer only asynchronous interfaces for indexDB.
```

Comment:
```suggestion
This is because browsers offer only asynchronous interfaces for IndexedDB.
```

Reply:
Wow I've been knowing the wrong name for years.